doc/source/user/troubleshooting.rst

```diff
@@ -0,0 +1,126 @@
+***************
+Troubleshooting
```

Comment:
Maybe make the title more specific, e.g. "Troubleshooting ImportError". The rest of the doc makes this immediately clear, but having it called out in the title might be nice.